plugins/SitesManager/lang/en.json

```diff
@@ -208,6 +208,9 @@
         "ConsentManagerConnected": "We’ve detected that %1$s is correctly configured to work with Matomo on your site.",
         "SiteWithoutDataReactDescription": "You can use the \"%1$sMatomo Tag Manager%2$s\" to integrate Matomo to track data, follow the instructions from this %3$sguide%4$s.",
         "ReactDetected": "React.js was detected on your website. Did you know you can use the \"%1$sMatomo Tag Manager%2$s\" to integrate Matomo into your site? %3$sLearn more in this guide%4$s.",
-        "ReactDetectedEmail": "React.js was detected on your website. Did you know you can use the \"Matomo Tag Manager\" to track the website data? Learn more here: %1$s"
+        "ReactDetectedEmail": "React.js was detected on your website. Did you know you can use the \"Matomo Tag Manager\" to track the website data? Learn more here: %1$s",
+        "SiteWithoutDataSPA": "SPA / PWA",
```

Comment:
I think this doesn't need to be translated. Those should be fixed abbreviations.

Reply:
Yes, removed from tranlsation